homeassistant/components/homekit/accessories.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, hass, driver, name, entity_id, aid, config,
         self._support_battery_charging = True
         self.linked_battery_sensor = \
             self.config.get(CONF_LINKED_BATTERY_SENSOR)
+        self.low_battery_threshold = \
+            self.config.get(CONF_LOW_BATTERY_THRESHOLD)
```

Comment:
Now that this configuration element has a default, we should use the indexor instead of `get`, as it will always be present:

```suggestion
            self.config[CONF_LOW_BATTERY_THRESHOLD]
```

Reply:
@cdce8p I think you inadvertently reverted this change?

Reply:
@andrewsayre That was on purpose. See: https://github.com/home-assistant/home-assistant/pull/23363#issuecomment-487413327